Directory.Build.props

```diff
@@ -11,6 +11,13 @@
     <!--This will target the latest patch release of the runtime released with the current SDK.  -->
     <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
     <RepositoryUrl>https://github.com/microsoft/healthcare-components</RepositoryUrl>
+
+    <SupportedFrameworks>netcoreapp3.1;net5.0</SupportedFrameworks>
+    <SdkPackageVersion>3.1.7</SdkPackageVersion>
```

Comment:
>3.1.7 [](start = 23, length = 5)

super nit: [https://versionsof.net/core/3.1/](https://versionsof.net/core/3.1/) latest one is `3.1.10` any reason to lock `3.1.7`?

Reply:
I guess answer is to keep PR small and to not introduce extra changes, right?

---
In reply to: [538782058](https://github.com/microsoft/healthcare-shared-components/pull/133#discussion_r538782058) [](ancestors = 538782058)

Reply:
That was my original intent, the only difference here should be the addition of .net5